### PR TITLE
Terrains - Regero Map Port 2, We Fixed the ROADS

### PR DESCRIPTION
**Pull request by killerswin2**
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:
 Added support for the Antistasi Mission on the Kingdom of Regero map by IceBreakr https://steamcommunity.com/sharedfiles/filedetails/?id=2278254320
Added Placeholder pictures for the port until better ones can be acquired. (<- that's for bob to get)
A redo of  #2195, this time Killer now knows what happened with the roads!

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### CODE REVIEW

**Review by Bob-Murphy — CHANGES_REQUESTED** [review]
You added a description.ext for Regero, but that contains the Chernarus Summer information.
You changed the Chernarus Summer description.ext over to Regero text.

Please reset the Chernarus Summer file and adapt the Regero one.

Please remove adding yourself to the debug console. I can add you, but that's the wrong way.

**Comment by jaj22** [comment]
Browsed this for a while. Main issue I see is that the vehicle places tend to be placed as if they were for a static park. These days they're used for spawning land QRFs and the land-vehicle portion of attacks, so they need to have reasonably straightforward access to a road (note that the direction of the marker matters). I was gonna list the bad ones, but that seemed to be a common trend, so give it a pass and then I'll re-check.

**Comment by killerswin2** [comment]
Yeah, this was designed back in fall of 2021

**Comment by killerswin2** [comment]
so, it does make sense that those places are static, given the context of the port

**Comment by jaj22** [comment]
It's not about making sense. It's how the mechanics work. Every vehicle placement is assumed to be accessible to a road, and if that's not obeyed then you'd end up with vehicles humping H-barriers and flying a kilometer into the air.

**Comment by killerswin2** [comment]
I wasn't condoning the liberal use of the static markers, no one ever told that they were wrong. But now you have brought it to my attention, so I will fix them.

**Comment by Bob-Murphy** [comment]
@killerswin2 I went through all the things. Please give it a spin

**Comment by jaj22** [comment]
@Bob-Murphy Did you redo the vehicle places?

**Comment by Bob-Murphy** [comment]
yes

File: A3A/addons/maps/mapInfo.hpp [hunk 1]
```diff
@@ -7,7 +7,7 @@ class mapInfo {
     #include "Antistasi_Enoch.Enoch\mapInfo.hpp"
     #include "Antistasi_Kunduz.Kunduz\mapInfo.hpp"
     #include "Antistasi_Malden.Malden\mapInfo.hpp"
-    #include "Antistasi_Regero.regero\mapInfo.hpp"
+    #include "Antistasi_regero.regero\mapInfo.hpp"
```
Comment (killerswin2) [inline A3A/addons/maps/mapInfo.hpp:10]: That's a path. Will need to rename the folder.

Comment (killerswin2) [reply A3A/addons/maps/mapInfo.hpp:10]: done

**Comment by killerswin2** [comment]
Good @Bob-Murphy

File: A3A/addons/maps/mapInfo.hpp (hunk 1 above)
Comment (Bob-Murphy) [reply A3A/addons/maps/mapInfo.hpp:10]: I renamed the folder on my end but it seems like Git was like `naaah bruv, I ain't carry the change over`.
Cheers

Comment (killerswin2) [reply A3A/addons/maps/mapInfo.hpp:10]: I did it for you, just pull the changes. If you're on windows it might not show up right.

**Pull request merged**